README.md

```diff
@@ -180,6 +180,47 @@ Providers
   </tr>
 </table>
 
+Example
+-------
+To generate a certificate for an apache2 website you can use code like this:
+
+    # Set up contact information. Note the mailto: notation
+    node.set['letsencrypt']['contact'] = [ 'mailto:me@example.com' ] 
+    # Real certificates please...
+    node.set['letsencrypt']['endpoint'] = 'https://acme-v01.api.letsencrypt.org' 
+
+    site="example.com"
+    sans=Array[ "www.#{site}" ]
+
+    # Set up your server here...
```

Comment:
I tried that first what I found out is taht I had to set up the web server configuraiton first, then create the self signed certificate and call an immediate restart after that to make sure that the webserver is actually running. You could do it the other way around, but then you would have to call an immediate restart every time your web server setup changes just to make sure the web server is running so I can renew/request the LetsEncrypt certificate.